### PR TITLE
feat(wasm): flag for setting contract admin

### PR DESCRIPTION
**Pull request by jcs47**

### CODE REVIEW

File: cosmwasm/utils.js [hunk 1]
```diff
@@ -66,8 +66,9 @@ const instantiateContract = (client, wallet, initMsg, config, { contractName, sa
                       initMsg,
                       contractName,
                       initFee,
+                      { admin: admin || account.address },
```
Comment (milapsheth) [inline cosmwasm/utils.js:69]: we should default the admin to the governance address if it's not overridden, not the deployer wallet

Comment (jcs47) [reply cosmwasm/utils.js]: done

File: cosmwasm/deploy-contract.js [hunk 2]
```diff
@@ -425,6 +425,7 @@ const programHandler = () => {
     program.addOption(new Option('-n, --chainNames <chainNames>', 'chain names').default('none'));
     program.addOption(new Option('-r, --reuseCodeId', 'reuse code Id'));
     program.addOption(new Option('-s, --salt <salt>', 'salt for instantiate2. defaults to contract name').env('SALT'));
+    program.addOption(new Option('--admin <address>', 'when instantiating contract, set an admin address other than the creator'));
```
Comment (milapsheth) [inline cosmwasm/deploy-contract.js:428]:
```suggestion
    program.addOption(new Option('--admin <address>', 'when instantiating contract, set an admin address other than the creator. Defaults to governance module account').default(governanceAddress));
```
and import it here

**Review by milapsheth — APPROVED** [review]

File: cosmwasm/deploy-contract.js (hunk 2 above)
Comment (jcs47) [reply cosmwasm/deploy-contract.js]: done

**Pull request merged**